packages/react-juce/src/components/Slider.tsx

```diff
@@ -0,0 +1,284 @@
+import React, { Component, PropsWithChildren } from 'react';
+
+import { Canvas } from './Canvas';
+import { View } from './View';
+import { SyntheticMouseEvent } from '../lib/SyntheticEvents';
+
+// Some simple helpers for slider drag gesture -> value mapping
+const _linearHorizontalGestureMap = (mouseDownX: number,
+                                     mouseDownY: number,
+                                     sensitivity: number,
+                                     valueAtDragStart: number,
+                                     dragEvent: SyntheticMouseEvent): number => {
+    let value = 0.0;
```

Comment:
Just for brevity, looks like we can omit this line, and collapse line 16 and 17 into a single `return Math.max(0, Math.min(1, ...etc))`?